TeachingRecordSystem/tests/TeachingRecordSystem.Api.UnitTests/GlobalUsings.cs

```diff
@@ -0,0 +1 @@
+//global using TeachingRecordSystem.TestCommon;
```

Comment:
Commented out to stop build warning; it'll be restored once we have some tests that use it.